src/components/DonationWidget/service.ts

```diff
@@ -0,0 +1,7 @@
+import { DonationMachineContext } from './machine/types';
+
+export const sendDonation = async (context: DonationMachineContext) => {
+  console.log('sendDonation', context);
```

Comment:
should this be here?

Reply:
For now yes, as this is a raw version for us to understand what's going on and there is no donation integration yet.